<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XVideo</class>
 <widget class="QWidget" name="XVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XVideo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#XVideo
{
background-color: rgb(68, 68, 68);
}
QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QLabel
{
	font: 75 12pt &quot;黑体&quot;;
	color: rgb(255, 255, 255);
}
    
#playButton
{
	background-color: rgba(255, 255, 255, 0);
}

#playButton:!hover
{
	image: url(:/new/prefix1/Resources/PlayN.png);
}

#playButton:hover
{
	image: url(:/new/prefix1/Resources/PlayH.png);
}

#playButton:pressed
{
	image: url(:/new/prefix1/Resources/PlayP.png);
}

#pauseButton
{
	background-color: rgba(255, 255, 255, 0);
}

#pauseButton:!hover
{
	image: url(:/new/prefix1/Resources/PauseN.png);
}

#pauseButton:hover
{
	image: url(:/new/prefix1/Resources/PauseH.png);
}

#pauseButton:pressed
{
	image: url(:/new/prefix1/Resources/PauseP.png);
}






</string>
  </property>
  <widget class="VideoWidget" name="src1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>412</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="open1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QSlider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度[0-100]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>对比度[1.0-3.0]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="birght">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="contrast">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="VideoWidget" name="des">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>400</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图像旋转</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rotate">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>450</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>270</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="flip">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>不处理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上下镜像</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>左右镜像</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上下左右镜像</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图像镜像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图像尺寸 宽和高</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="width">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>570</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图像金字塔（高斯、拉普拉斯）</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pyup">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="pydown">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>水印(x,y,a)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="mx">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="my">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ma">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>XVideo</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>XVideo</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>XVideo</receiver>
   <slot>SetPos(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open1</sender>
   <signal>clicked()</signal>
   <receiver>XVideo</receiver>
   <slot>Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideo</receiver>
   <slot>Set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideo</receiver>
   <slot>Export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideo</receiver>
   <slot>Play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideo</receiver>
   <slot>Pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton_2</sender>
   <signal>clicked()</signal>
   <receiver>XVideo</receiver>
   <slot>Mark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>-69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open()</slot>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
  <slot>SetPos(int)</slot>
  <slot>Set()</slot>
  <slot>Export()</slot>
  <slot>Play()</slot>
  <slot>Pause()</slot>
  <slot>Mark()</slot>
 </slots>
</ui>
